### ISSUE TITLE
The package is incompatible with angular 13

### ISSUE THREAD

**Issue by orgads** [ngx-formly/ngx-formly#3049]
**Description**
When `@angular/core@13` is installed, `require @ngx-formly/core` fails with the following error:
```
node:internal/modules/cjs/loader:979
    throw new ERR_REQUIRE_ESM(filename, true);
    ^

Error [ERR_REQUIRE_ESM]: require() of ES Module F:\Projects\test-formly\node_modules\@angular\common\fesm2015\common.mjs not supported.
Instead change the require of F:\Projects\test-formly\node_modules\@angular\common\fesm2015\common.mjs to a dynamic import() which is available in all CommonJS modules.
    at F:\Projects\test-formly\node_modules\@ngx-formly\core\bundles\ngx-formly-core.umd.js:2:85
    at Object.<anonymous> (F:\Projects\test-formly\node_modules\@ngx-formly\core\bundles\ngx-formly-core.umd.js:5:2)
    at Object.<anonymous> (F:\Projects\test-formly\test.js:1:1) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Minimal Reproduction**
[bug3049.zip](https://github.com/ngx-formly/ngx-formly/files/7533733/bug3049.zip)

**Your Environment**

- Angular version: 13.0.1
- Formly version: 5.10.23

**Comment by aitboudad** [comment]
could you please check with https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-next.1

**Comment by orgads** [comment]
It's the same.

**Comment by orgads** [comment]
The reproduction is extremely simple:

```
npm i @ngx-formly/core @angular/core@13
node -r @ngx-formly/core
```

**Comment by aitboudad** [comment]
Ok I'll check on my side once I've got the chance

**Comment by aitboudad** [comment]
Angular v13 support is released in [v6.0.0-next.4 (release)](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-next.4), to give it a try use `next` tag:

```
ng add @ngx-formly/schematics@next --ui-theme=bootstrap
```
